src/sample/SQL/CAT SQL.sql

```diff
@@ -0,0 +1,83 @@
+CREATE DATABASE COURSE_REGISTER;
+USE COURSE_REGISTER;
+
+CREATE TABLE COURSE(
+ COURSE_ID CHAR(6) NOT NULL,
+ COURSE_NAME VARCHAR(100) NOT NULL,
+ PRIMARY KEY(COURSE_ID)
+ );
+ 
+ CREATE TABLE STUDENT(
+  MATRIC_NUMBER CHAR(6) NOT NULL,
+  STU_NAME VARCHAR(30) NOT NULL,
+  STU_IC_NUM CHAR(5) NOT NULL,
+  ACADDEMIC_STATUS VARCHAR(8) NOT NULL,
+  SEMESTER_REGISTERED CHAR(3) NOT NULL,
+  STU_CGPA DECIMAL(3,2) NOT NULL,
+  STU_YEAR CHAR(1) NOT NULL,
+  STU_SCHOOL VARCHAR(50) NOT NULL,
+  STU_CAMPUS VARCHAR(50) NOT NULL,
+  STU_PROGRAMME VARCHAR (50) NOT NULL,
+  PRIMARY KEY(MATRIC_NUMBER)
+  );
+  
+  CREATE TABLE LECTURER(
+   STAFF_ID CHAR(6) NOT NULL,
+   LECT_NAME VARCHAR(30) NOT NULL,
+   LECT_IC_NUMBER CHAR(5) NOT NULL,
+   LECT_STATUS VARCHAR(8) NOT NULL,
+   LECT_SCHOOL VARCHAR(50) NOT NULL,
+   LECT_POSITION VARCHAR(50) NOT NULL,
+   PRIMARY KEY(STAFF_ID)
+   );
+ 
+
+   
+CREATE TABLE STUDENT_ACCOUNT(
+ STU_EMAIL VARCHAR(319) NOT NULL,
+ STU_PASSWORD VARCHAR(100) NOT NULL,
+ MATRIC_NUMBER CHAR(6) NOT NULL,
+ FOREIGN KEY(MATRIC_NUMBER) 
+	REFERENCES STUDENT (MATRIC_NUMBER) ON DELETE CASCADE
+ );
+ 
+ CREATE TABLE COURSE_STUDENT_LECT(
+  COURSE_ID CHAR(6) NOT NULL,
+  MATRIC_ID CHAR(6) NOT NULL,
+  STAFF_ID CHAR(6) NOT NULL,
+  STUDENT_COURSE_STATUS CHAR(1) NOT NULL,
+  FOREIGN KEY (STAFF_ID) 
+	REFERENCES LECTURER (STAFF_ID) ON DELETE CASCADE
+  );
+
+ CREATE TABLE LECTURER_ACCOUNT(
+	LECT_EMAIL VARCHAR(319) NOT NULL,
+    LECT_PASSWORD VARCHAR(100) NOT NULL,
+    STAFF_ID CHAR(6) 
+     REFERENCES LECTURER (STAFF_ID) ON DELETE CASCADE
+    );
+
+
+INSERT INTO STUDENT VALUES (123456, 'Heng Heng', 52312, 'ACTIVE', 201, 3.77, 1, 'Computer Science', 'Main Campus', 'Computer Science');
```

Comment:
Student table, matric_num is chat type so the 123456 should be '123456' but if you change the type to numbers, or int, then this can remain